x-pack/plugins/observability/public/pages/alerts/alerts.test.tsx

```diff
@@ -60,7 +60,6 @@ jest.spyOn(pluginContext, 'usePluginContext').mockImplementation(() => ({
   } as unknown as AppMountParameters,
   config: {
     unsafe: {
-      slo: { enabled: false },
```

Comment:
So the whole slo feature was behind a feature flag `xpack.observability.unsafe.slo.enabled`? I understand that you deleted the `xpack.observability.unsafe.alertDetails.slo.enabled` feature flag that you introduced in this PR, but I don't get it why you remove this feature flag. Was it a leftover? Should we document it in a READMe file?

Reply:
This was an artifact of the time slo was behind a feature flag indeed: `xpack.observability.unsafe.slo.enabled`

As for `xpack.observability.unsafe.alertDetails.slo.enabled`, I thought initially to use a feature flag, but then I did everything in this PR so no need for it. But it's possible I've forgotten one somewhere.

Reply:
Also, this feature flag removal (slo.enabled) is an artifact from the test only. That's because the test did not use the proper Config type, therefore the leftover never triggered a type error. But this config property is not present anymore